custom_components/pyscript/trigger.py

```diff
@@ -22,7 +22,7 @@
 _LOGGER = logging.getLogger(LOGGER_PATH + ".trigger")
 
 
-STATE_RE = re.compile(r"[a-zA-Z]\w*\.[a-zA-Z]\w*$")
+STATE_RE = re.compile(r"[a-zA-Z]\w*\.[a-zA-Z]\w*(\.[a-zA-Z\*]?[a-zA-Z]*)?$")
```

Comment:
This RE isn't quite right, since it allows `x.y.*attr` and doesn't allow underscores or digits in the attribute name.

Here's a corrected version:
```
STATE_RE = re.compile(r"[a-zA-Z]\w*\.[a-zA-Z]\w*(\.(([a-zA-Z]\w*)|\*))?$")
```